### PR TITLE
[Issue 633] Add Alsia and Michael to project

### LINKED ISSUE #633: [Task]: Add Michael to project team and code maintainers file
### Summary

@chouinar has joined the project as a senior eng.

### Acceptance criteria

- [ ] Michael is added to project team and code maintainers file

### PR DESCRIPTION
**Pull request by acouch**
## Summary
Fixes #633

## Changes proposed
Add new team members to MAINTAINERS and CODEOWNERS

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # We are limiting this to 3 engineers and choosing not to select "Require review from Code Owners."
-@daphnegold @SammySteiner @acouch
+@acouch @aplybeah @chouinar @SammySteiner
```
Comment (chouinar) [inline .github/CODEOWNERS:2]: Should we adjust this comment?

File: docker-compose-ssl.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: '3'
```
Comment (chouinar) [inline docker-compose-ssl.yml:1]: What is this file?

**Review by chouinar — COMMENTED** [review]
Is the Codeowners file working? Shouldn't the people automatically be added as reviewers when we open a PR - I've had to manually add people

File: docker-compose-ssl.yml (hunk 2 above)
Comment (acouch) [reply docker-compose-ssl.yml:1]: Added by oopsies. Removing.

**Comment by acouch** [comment]
@chouinar thanks for the review. Per yr question, it should auto-assign. The file is "valid":

image

🤔

**Comment by chouinar** [comment]
> @chouinar thanks for the review. Per yr question, it should auto-assign. The file is "valid":
> 
> image
> 
> 🤔

We might need to make it so a code owner needs to approve the PR to merge for it to automatically work?

**Review by chouinar — APPROVED** [review]
Approving - although we should also adjust the repo to use the file for assigning automatically

**Pull request merged**